src/design-system/whats-changed.md.njk

```diff
@@ -13,6 +13,245 @@ layout: layout-single-page-prose.njk
 
 The tables below show the old and new names for components, classes and mixins, to help you find what you need.
 
+## Template
+
+Previously to keep your pages consistent with the rest of GOV.UK you would have used [GOV.UK Template](http://alphagov.github.io/govuk_template/).
+
+We have brought this into GOV.UK Frontend and it can be consumed via Nunjucks or HTML.
+
+If you are not using Nunjucks you can get the HTML from the [Template guidance page](../styles/template/).
+
+### Nunjucks
+
+Where possible we have kept the Template API the same, with the main changes being camelCase vs snake_case.
```

Comment:
Suggest changing to 'Where possible we have kept the Template API the same. The main change is from using snake_case to camelCase.'

Do snake_case and camelCase need describing for anyone? If so, link to guidance.